aspnetcore/common/samples/WebApplication1/Startup.cs

```diff
@@ -75,7 +75,29 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
 
             app.UseIdentity();
 
-            // Add external authentication middleware below. To configure them please see http://go.microsoft.com/fwlink/?LinkID=532715
+            app.UseFacebookAuthentication(new FacebookOptions()
```

Comment:
@anathemort do you mean that everything but Facebook should be removed? Or remove everything including Facebook?

@Tratcher what do you think? If that sample the social login topics are referring to is used as-is to demonstrate a basic Core app, anyone who tries to run it will fail if they don't set the secrets. I can remove code links and replace with inline code.